package.json

```diff
@@ -6,19 +6,24 @@
     "eject": "expo eject",
     "ios": "expo start --ios",
     "start": "expo start",
-    "test": "jest",
+    "test": "jest --verbose --coverage",
     "web": "expo start --web"
   },
   "dependencies": {
     "@react-native-community/masked-view": "0.1.5",
```

Comment:
Unrelated to your change, I think we should remove "@react-native-community/masked-view", since it isn't being used and the only way we can use it is by ejecting (https://www.npmjs.com/package/@react-native-community/masked-view)

what do you think @bit172

Reply:
Yes, since it's not being used we can remove it.